benches/boringssl/kyber768.cxx

```diff
@@ -1,4 +1,4 @@
-#include <string>
+#include <cstring>
```

Comment:
We could just remove this line altogether

Reply:
sure, if it's not used ... let's see what the CI says.
Also there's something off on the CI. It works on main but fails here. I have to look into that.

Reply:
It's needed, you use memset

Reply:
Strange that I can build locally without the header, but oh well